docs/tuning-apps/fault-tolerance/supported.rst

```diff
@@ -0,0 +1,39 @@
+Supported fault tolerance techniques
+====================================
+
+Open MPI is a vehicle for research in fault tolerance and over the years
+provided support for a wide range of resilience techniques:
+
+* Current
+    * User Level Fault Mitigation techniques similar to
+      those defined in the context of the MPI Forum (this is the closest match when migrating from FT-MPI);  :ref:`See its documentation section <ulfm-label>`.
+
+* Deprecated / no longer available
+    * Coordinated and uncoordinated process checkpoint and
+      restart. Similar to those implemented in LAM/MPI and MPICH-V,
+      respectively.
+    * |strikethru_start| Message logging techniques. Similar to those
+      implemented in MPICH-V |strikethru_end|
+    * |strikethru_start| Data Reliability and network fault tolerance. Similar
+      to those implemented in LA-MPI |strikethru_end|
```

Comment:
I don't know, they were there in your original conversion of the file to RST: https://github.com/open-mpi/ompi/commit/e4db776046cbfd0bad3c17942bcf09103c4fe8f5#diff-58f725531a70e71f53629d75173f2fe204dd7fb393584762fce5b64526545fae

Reply:
I've tried to determine where that info came from before your commit, but I can't find a pattern in the new RST file that is in the repo before your commit.   Was that file added new for the switchover to RST?

Reply:
Looks like you pulled it from faq/fault-tolerance.rst (where it used `<strike>` / `</strike>`, not the new macros).

@abouteiller What does it mean that these 2 items are struck out in the FT doc?

Reply:
Yes, that's exactly true.  However, I don't know why they are crossed out originally.  If that's needed, I could use help to determine what original file was used as input for creating faq/fault-tolerance.rst...  Hopefully @abouteiller can tell us... :-)

Reply:
The features are not available for general use
1. at all, for the case of data reliability (this has been removed when the PML DR got removed d692aba6 )
2. not suitable for production use, only for research (pml/v component)

Reply:
Ok.  

* For 1, can we remove the bullet altogether?
* For 2, can we not strike it out, but rather say that it's only for research usage, ... etc.?

I think the strikeout is ambiguous as to what it actually means.

Once the strikout is removed, those 2 new macros should be removed, too.

Reply:
@abouteiller In an effort to keep this PR moving, I pushed a 3rd commit as a suggestion for this FT content.  Can you review?  If acceptable, all the commits in this PR should be squashed so that we can merge this PR.